common/lib/xmodule/xmodule/js/src/video/01_initialize.js

```diff
@@ -693,7 +701,7 @@ function (VideoPlayer, i18n) {
 
     function getDuration() {
         try {
-            return this.metadata[this.youtubeId()].duration;
+            return moment.duration(this.metadata[this.youtubeId()].duration, moment.ISO_8601).asSeconds();
```

Comment:
There are existing test around this function https://github.com/edx/edx-platform/pull/8636/files#diff-29ae755a37b293a4c4e99723b2ba5645R68. I've already updated them accordingly.